not-to-release/sources/email/enronsent24_01.xml.conllu

```diff
@@ -153,7 +153,7 @@
 11	revision	revision	NOUN	NN	Number=Sing	8	obj	8:obj	_
 12	concerning	concern	VERB	VBG	VerbForm=Ger	16	case	16:case	_
 13	Section	section	NOUN	NN	Number=Sing	16	compound	16:compound	_
-14	XIII	xiii	NUM	CD	NumType=Card	13	nummod	13:nummod	_
+14	XIII	xiii	NUM	CD	NumForm=Roman|NumType=Card	13	nummod	13:nummod	_
```

Comment:
lemma should be capitalized

Reply:
Alright, updated all of the lemmas to match the original text.  Look good?

Reply:
Yes!